vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh

```diff
@@ -17,7 +19,12 @@ do
     fi
 done
 
-sig_resource_id="/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP_NAME}/providers/Microsoft.Compute/galleries/PackerSigGalleryEastUS/images/1804Gen2/versions/1.0.${CREATE_TIME}"
+IMAGENAME=${OS_VERSION//./}Gen2
+if [[ $OS_SKU != "Ubuntu" ]]; then
+IMAGENAME=${OS_SKU}${IMAGENAME}
+fi
```

Comment:
we should let caller pass in the right image name. This util script should be as general as it can be